scripts/xe-reset-networking

```diff
@@ -85,12 +85,22 @@ if __name__ == "__main__":
 		try:
 			conf = read_management_conf()
 			device = conf['LABEL']
+                        # Get the VLAN if provided in the firstboot data and not specified by the user
+                        if options.vlan == None:
```

Comment:
I don't think that is wrong. They may choose to not change the VLAN, but use another NIC. In any case, before the script does anything at all, it will display a summary so that the user can confirm before proceeding.